src/HoldRequestResultConsumerListener.php

```diff
@@ -254,15 +252,15 @@ protected function skipMissingPatron(HoldRequestResult $holdRequestResult)
 
     /**
      * @param Patron $patron
-     * @param Bib $bib
-     * @param Item $item
+     * @param array $bibs
```

Comment:
@gkallenberg @kfriedman I've changed the type hinting of $bibs to array here. Also poking around to see if type hinting is needed elsewhere.